pkg/rhos/gw-machineset.go

```diff
@@ -66,12 +66,11 @@ spec:
                 name: {{.InfraID}}-nodes
                 tags: openshiftClusterID={{.InfraID}}
           securityGroups:
-          - filter: {}
-            name: {{.InfraID}}-worker
-          - filter: {}
-            name: {{.InfraID}}-submariner-gw-sg
-          - filter: {}
-            name: {{.InfraID}}-submariner-internal-sg
+          - name: {{.InfraID}}-worker
+          {{- if .AddSubmarinerInternalSG }}
```

Comment:
Wouldn’t it be more accurate to call this `UseSubmarinerInternalSG` here? Does setting *this* field actually result in the SG being added?

Reply:
Done.